src/agents/usage.py

```diff
@@ -70,13 +81,22 @@ def __post_init__(self) -> None:
         if self.output_tokens_details.reasoning_tokens is None:
             self.output_tokens_details = OutputTokensDetails(reasoning_tokens=0)
 
-    def add(self, other: "Usage") -> None:
+    def add(
+        self,
+        other: Usage,
+        model_name: str,
```

Comment:
these properties are in the request usage data, so i don't think these data should be passed this way. at least, we should avoid passing these in order to keep flexibility of future enhancement (meaning having additional-properties argument, which can accept more properties in the future)